examples/topic-aliasing/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import { ExtensionContext } from "@foxglove/studio";
+
+export function activate(extensionContext: ExtensionContext): void {
+  // Register a topic alias function that takes the current list of datasource topics and
+  // global variables and outputs a list of topic aliases.
+  extensionContext.registerTopicAliases(({ topics, globalVariables }) => {
+    // Output a list of aliased topics, in this case influenced by the current value of
+    // the global variable `device`.
+    const device = globalVariables["device"] ?? "default";
```

Comment:
Maybe we use the "camera_name" example where we have a variable that is the camera name from our sample data? This way its a bit more interesting example?